src/librustc_metadata/diagnostics.rs

```diff
@@ -14,14 +14,14 @@ register_long_diagnostics! {
 E0454: r##"
 A link name was given with an empty name. Erroneous code example:
```

Comment:
"Erroneous code example" should be on "its own line" (don't see better way to say it ^^').